securitytxt-signer.sh

```diff
@@ -33,6 +33,10 @@ if [ "$#" -lt 1 ]; then
   exit 1
 fi
 
+# Prepare environment
+
+[[ "$(uname)" == "Darwin" ]] && IS_MAC=true || IS_MAC=false
```

Comment:
Bash doesn't really have booleans. If you used `1` &  `0` instead, you could use arithmetic expressions.

```suggestion
[[ "$(uname)" == "Darwin" ]] && IS_MAC=1 || IS_MAC=0
```

Then:

```
$ IS_MAC=1 ; ((IS_MAC)) && echo "true" || echo "false"
true
$ IS_MAC=0 ; ((IS_MAC)) && echo "true" || echo "false"
false

Reply:
On the other hand we should consider [SC2015](https://www.shellcheck.net/wiki/SC2015):

> Note that `A && B || C` is not if-then-else. C may run when A is true.

This might require a full if-then-else.

Reply:
If `uname` on macOS has `-s`, this could explicitly be `$(uname -s)` 🤔, but that is a minor improvement.